Lib/glyphsLib/builder/builders.py

```diff
@@ -205,7 +205,7 @@ def designspace(self):
         self.to_designspace_family_user_data()
 
         # append base style shared by all masters to designspace file name
-        base_family = self.font.familyName or 'Unnamed'
+        base_family = self.family_name or 'Unnamed'
```

Comment:
note that this `self.family_name` attribute defaults to the `self.font.familyName` when the `family_name` argument is not provided in the UFOBuilder constructor, so this continues to work the same as before if one does not use the fontmake `--family-name` option.